home/tests/test_content_import_export.py

```diff
@@ -1674,6 +1674,17 @@ def test_hidden_characters(self, csv_impexp: ImportExport) -> None:
         assert "\u2028\u2028" in csv_bytes.decode("utf-8")
         assert "\u2028" not in ContentPage.objects.all().values()[0]["whatsapp_body"]
 
+    def test_list_item_descriptive_error_message(
+        self, csv_impexp: ImportExport
+    ) -> None:
+        """
+        Import failure for list_items should return a descriptive error message
+        """
```

Comment:
Can we try add a test for a different kind of error to make sure that code path works as expected?